From 42593e7230fad9f87732c1687cb5daff918f2fa1 Mon Sep 17 00:00:00 2001
From: Colin Watson <cjwatson@ubuntu.com>
Date: Wed, 26 Sep 2012 13:51:13 +0100
Subject: [PATCH 049/364] * Makefile.util.def (grub-mknetdir): Move to
 $prefix/bin. Reported by: Daniel Kahn Gillmor.  Fixes Debian bug #688799.

---
 ChangeLog         | 5 +++++
 Makefile.util.def | 4 ++--
 2 files changed, 7 insertions(+), 2 deletions(-)

diff --git a/ChangeLog b/ChangeLog
index c43f8a4..2658573 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,5 +1,10 @@
 2012-09-26  Colin Watson  <cjwatson@ubuntu.com>
 
+	* Makefile.util.def (grub-mknetdir): Move to $prefix/bin.
+	Reported by: Daniel Kahn Gillmor.  Fixes Debian bug #688799.
+
+2012-09-26  Colin Watson  <cjwatson@ubuntu.com>
+
 	* util/grub-mkconfig_lib.in (is_path_readable_by_grub): Redirect
 	errors from grub-probe to /dev/null, not stdout.
 
diff --git a/Makefile.util.def b/Makefile.util.def
index 72057cf..8324ede 100644
--- a/Makefile.util.def
+++ b/Makefile.util.def
@@ -487,8 +487,8 @@ script = {
 };
 
 script = {
-  mansection = 8;
-  installdir = sbin;
+  mansection = 1;
+  installdir = bin;
   name = grub-mknetdir;
 
   common = util/grub-mknetdir.in;
-- 
1.8.1.4

